inspirehep/modules/workflows/workflows/article.py

```diff
@@ -76,6 +76,7 @@
 from inspirehep.modules.workflows.tasks.upload import store_record, set_schema
 from inspirehep.modules.workflows.tasks.submission import (
     add_note_entry,
+    add_internal_uid,
```

Comment:
Why not add it directly to https://github.com/jacquerie/inspire-next/blob/3316f65c3fe9b60c68c8c59e9491a1564f431986/inspirehep/modules/literaturesuggest/tasks.py#L206? Even though `formdata_to_model` might need a refactor, for the moment it would be easier to reason if adding `acquisition_source` is in one place.